lib/index.d.ts

```diff
@@ -0,0 +1,644 @@
+/// <reference types="@panva/jose" />
+/// <reference types="node" />
+
+/**
+ * @see https://github.com/panva/node-openid-client/blob/master/docs/README.md
+ */
+declare module 'openid-client' {
+  import { JWKS, JSONWebKeySet } from '@panva/jose'
+  import { IncomingMessage } from 'http'
+  import { Http2ServerRequest } from 'http2'
+  import { GotOptions } from 'got'
+
+  type HttpRequestOptions = GotOptions<null>
+  type CustomHttpOptionsProvider = (options: HttpRequestOptions) => HttpRequestOptions
+
+  /**
+   * @see https://github.com/panva/node-openid-client/blob/master/lib/index.js
+   */
+  export const custom : {
+    setHttpOptionsDefaults(params: ISetHttpOptionsDefaults) : void
```

Comment:
params here is actually following the `GotOptions` interface too

Reply:
ok